src/DssSpell.sol

```diff
@@ -37,7 +39,7 @@ interface DssVestLike {
   ) external returns (uint256);
 }
 
-contract DssSpellAction is DssAction {
+contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
```

Comment:
This is probably all fine, and I don't want to bikeshed too much, but what about just going with `DssSpellCollateralAction` since CES (and we) might want to do more than just onboarding in this file. Might need to reconsider the name of the execution function though.

Reply:
I went with onboarding because i didn't want collateral line, dust, or rate limits getting put in there, i saw it as just onboarding collateral.  However, if we wanted to expand it and have a "collateral" spell that could work too. lmk